annotation-processing/src/main/java/com/exactpro/epfast/annotation/processing/helpers/FastPackageNameEncoder.java

```diff
@@ -0,0 +1,15 @@
+package com.exactpro.epfast.annotation.processing.helpers;
+
+public class FastPackageNameEncoder {
+    public static String encode(String rawPath) {
+        String[] packages = rawPath.split("\\.");
```

Comment:
It can be done simpler with streams:
```java
        return Arrays.stream(rawPath.split(Pattern.quote(".")))
            .map(x -> x + "$").collect(Collectors.joining("."));
```